services/core-web/src/components/noticeOfWork/NOWProgressActions.js

```diff
@@ -168,7 +165,6 @@ export class NOWProgressActions extends Component {
   handleStartDelay = (values) => {
     const payload = {
       ...values,
-      start_date: new Date().toISOString(),
     };
```

Comment:
You can just pass `values` into the `createApplicationDelay` instead of spreading them into a new const

Reply:
changed